prime-angular-frontend/src/app/shared/classes/dialog-logger.ts

```diff
@@ -0,0 +1,36 @@
+import { Injectable } from '@angular/core';
+import { MatDialog } from '@angular/material/dialog';
+
+import { ConfirmDialogComponent } from '@shared/components/dialogs/confirm-dialog/confirm-dialog.component';
+import { ErrorLoggerComponent } from '@shared/components/dialogs/content/error-logger/error-logger.component';
+import { DialogOptions } from '@shared/components/dialogs/dialog-options.model';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class DialogLogger {
+  constructor(
+    private dialog: MatDialog
+  ) { }
+
+  public log(error: Error): void {
```

Comment:
The messaging on this dialog is a bit hard for users to understand.  We need a generic message indicating an "unknown error" has occurred at a "date and time", and provide a unique error ID they can pass off to a "specific contact".  I provided Pilar with the quoted strings to construct the message with Sarah.  

Can you update the API logger to respond with the ID of the row it just added and we'll use that as the unique error ID to help locate errors easier when reported, and then it can be passed as a parameter to `log`.